src/main/java/fr/litopia/model/Velo.java

```diff
@@ -24,7 +24,7 @@
     @OneToOne(mappedBy = "velo")
     private Bornette bornette;
 
-    @OneToMany(mappedBy = "velo")
+    @OneToMany(mappedBy = "velo",cascade = CascadeType.ALL)
```

Comment:
```suggestion
    @OneToMany(mappedBy = "velo")
```
Je vais tenter de régles le bug moi même peut être pas comme sa.